UPGRADE-4.0.md

```diff
@@ -16,6 +16,26 @@ Form
 
  * Support for data objects that implements both `Traversable` and
    `ArrayAccess` in `ResizeFormListener::preSubmit` method has been removed.
+   
+ * Using callable strings as choice options in ChoiceType has been deprecated
+   in favor of `PropertyPath`. Use a "\Closure" instead
```

Comment:
The sentence as is must be added to the `UPGRADE-3.1.md` file. In this file it should read like "Using callable strings as choice options in ChoiceType has been removed in favor of passing `PropertyPath` instances."